cmd/backup/main.go

```diff
@@ -273,6 +275,12 @@ func newScript() (*script, error) {
 		})
 	}
 
+	tmpl, err := template.ParseGlob("/etc/volume-backup.d/*notifications.tmpl")
```

Comment:
Two questions here:
- should this maybe use `/etc/volume-backup/notifications.d` so it's more future proof (maybe some other feature wants to add something to `/etc/volume-backup` at some point
- should we just parse all files in this directory, no matter the naming? This `00foo.tmpl` is just a convention that is used often, but I think in theory most tools will accept everything.

Reply:
If we want to be super consistent: the lockfile is currently named `"/var/lock/dockervolumebackup.lock"` so maybe this should be `/etc/dockervolumebackup/notifications.d/*.*`?

Reply:
Another idea here would be to embed the default template in the binary itself and only read overrides from that dir:

```go
	var tmplErr error
	tmpl := template.New("notifications")
	tmpl, tmplErr = tmpl.Parse(defaultNotifications)
	if tmplErr != nil {
		return nil, fmt.Errorf("newScript: unable to parse default notifications templates: %w", err)
	}

	if _, err := os.Stat("/etc/dockervolumebackup/notifications.d"); err == nil {
		tmpl, tmplErr = tmpl.ParseGlob("/etc/dockervolumebackup/notifications.d/*.*")
		if tmplErr != nil {
			return nil, fmt.Errorf("newScript: unable to parse user defined notifications templates: %w", err)
		}
	}

	s.template = tmpl
```

The embed would happen at top level then:

```go
import _ "embed"

//go:embed notifications.tmpl
var defaultNotifications string
```